### PR TITLE
Android - Reinforcement learning - Plane strike game

### PR DESCRIPTION
**Pull request by st-haingo**
- [x] Using `Interpreter`
- [x] Using Jetpack compose/ kotlin flow

**GIF**

### CODE REVIEW

**Comment by st-haingo** [comment]
Will update readme soon..

File: examples/reinforcement_learning/app/src/main/java/com/google/edgeai/examples/reinforcement_learning/ReinforcementLearningHelper.kt [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+class ReinforcementLearningHelper(private val context: Context) {
+    /** The TFLite interpreter instance.  */
+    private var interpreter: Interpreter? = null
+
+    init {
+        initHelper()
+    }
+
+    val hit: SharedFlow<Int>
```
Comment (PaulTR) [inline examples/reinforcement_learning/app/src/main/java/com/google/edgeai/examples/reinforcement_learning/ReinforcementLearningHelper.kt:37]: Doesn't look like this hit value is used. Can you delete and confirm that everything still works on your side?

File: examples/reinforcement_learning/app/src/main/java/com/google/edgeai/examples/reinforcement_learning/ReinforcementLearningHelper.kt [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    private val _hit = MutableSharedFlow<Int>()
+
+    /** Init a Interpreter from asset*/
+    private fun initHelper() {
+        interpreter = try {
+            val tfliteBuffer = FileUtil.loadMappedFile(context, "planestrike.tflite")
+            Log.i(TAG, "Done creating TFLite buffer from asset")
+            Interpreter(tfliteBuffer, Interpreter.Options())
+        } catch (e: Exception) {
+            Log.i(TAG, "Create TFLite is failed ${e.message}")
```
Comment (PaulTR) [inline examples/reinforcement_learning/app/src/main/java/com/google/edgeai/examples/reinforcement_learning/ReinforcementLearningHelper.kt:48]: Let's change the error message to "Initializing TensorFlow Lite has failed with error: ${e.message}"

**Comment by PaulTR** [comment]
Please change the directory so the root is reinforcement_learning/android

**Pull request merged**